website/docs/plugin/hashicorp-provider-design-principles.mdx

```diff
@@ -29,7 +29,7 @@ The benefits of this practice include:
 
 ## Resource and attribute schema should closely match the underlying API
```

Comment:
We should consider moving this "Provider Design" page under the Best Practices section as well.